src/components/vm/vmDetailsPage.jsx

```diff
@@ -88,17 +88,15 @@ export const VmDetailsPage = ({
     if (cockpit.location.path[1] == "console") {
         return (
             <WithDialogs key="vm-details">
-                <Page id={"vm-" + vm.name + "-consoles-page"}
-                      className="consoles-page-expanded no-masthead-sidebar">
-                    <PageSection hasBodyWrapper={false}>
-                        <ConsoleCard
-                            state={consoleState}
-                            vm={vm}
-                            config={config}
-                            onAddErrorNotification={onAddErrorNotification}
-                            isExpanded />
-                    </PageSection>
-                </Page>
+                <div id={"vm-" + vm.name + "-consoles-page"}
+                     className="consoles-page-expanded">
+                    <ConsoleCard
+                        state={consoleState}
+                        vm={vm}
+                        config={config}
+                        onAddErrorNotification={onAddErrorNotification}
+                        isExpanded />
+                </div>
```

Comment:
These 9 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2229-eaaa9181-20250728-131638-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)